purchasing/static/js/shared/extlink.js

```diff
@@ -0,0 +1,6 @@
+$(document).ready(function() {
+    $(".external").tooltip({
```

Comment:
I would prefer a more specific selector, and one that includes some indication that it's being manipulated by javascript.

For example: `js-external-link` or something like that.